src/MatchedLearnerApi.Application/Data/ModelConfiguration/DatalockEventNonPayablePeriodFailureConfiguration.cs

```diff
@@ -0,0 +1,18 @@
+﻿using MatchedLearnerApi.Application.Data.Models;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace MatchedLearnerApi.Application.Data.ModelConfiguration
+{
+    public class DatalockEventNonPayablePeriodFailureConfiguration : IEntityTypeConfiguration<DatalockEventNonPayablePeriodFailure>
```

Comment:
All the data lock stuff is called "Datalock" but the table in the DB, and all references to data locks in the payments repo call it "DataLock" should we make this change now before we introduce an in-consistency?

Reply:
I will make this change when in the big refactor branch